app/js/components/orderEntry/SearchAndAddOrder.jsx

```diff
@@ -4,62 +4,73 @@ import axios from 'axios';
 import PropTypes from 'prop-types';
 import ActiveOrders from './ActiveOrders';
 import PastOrders from './PastOrders';
+import { withRouter } from 'react-router-dom';
 import AddForm from './addForm/AddForm';
 import Tabs from '../tabs/Tabs';
 import Tab from '../tabs/Tab';
-import {
-  fetchInpatientCareSetting,
-  fetchOutpatientCareSetting,
-} from '../../actions/careSetting';
-import SearchDrug from '../searchDrug';
+import { getPastOrders } from '../../actions/pastOrders';
 
 export class SearchAndAddOrder extends React.Component {
   constructor(props) {
-    super();
+    super(props);
     this.state = {
       careSetting: 'OutPatient',
+      uuid: '',
     };
   }
 
   componentDidMount() {
-    this.props.fetchInpatientCareSetting();
-    this.props.fetchOutpatientCareSetting();
+    const query = new URLSearchParams(this.props.location.search);
+    const uuid = query.get('patient');
+    this.props.getPastOrders(uuid, this.props.outpatientCareSetting.uuid);
+    this.setState({ uuid: query.get('patient') });
   }
 
-
   handleCareSettings = (careSetting) => {
     this.setState({ careSetting });
+    if (careSetting === 'InPatient') {
+      this.props.getPastOrders(this.state.uuid, this.props.inpatientCareSetting.uuid);
+    } else {
+      this.props.getPastOrders(this.state.uuid, this.props.outpatientCareSetting.uuid);
+    }
   }
+
   render() {
     return (
       <div className="body-wrapper">
         <Tabs careSetting={this.handleCareSettings}>
           <Tab
-            tabName="Outpatient">
-            <SearchDrug />
+            tabName="OutPatient">
+            <form className="sized-inputs css-form" noValidate>
+              <p className="input-position-class">
+                <label name="input-id">Input label</label>
+                <input id="input-id" />
+              </p>
+            </form>
+            <PastOrders pastOrders={this.props.pastOrders} />
           </Tab>
-          <Tab tabName="Inpatient">
-            <SearchDrug />
+          <Tab tabName="InPatient">Tab 2 content
+            <PastOrders pastOrders={this.props.pastOrders} />
+
           </Tab>
         </Tabs>
       </div>
     );
   }
 }
 
-const mapStateToProps = ({ careSettingReducer }) => ({
+const mapStateToProps = ({ careSettingReducer, pastOrdersReducer }) => ({
```

Comment:
You could further destructure inpatientCareSetting and outpatientCareSetting from state. This will offer the benefit of using es6 object notation like below
```
const mapStateToProps = ({ careSettingReducer : {inpatientCareSetting, outpatientCareSetting}, pastOrdersReducer }) => ({
inpatientCareSetting,
outpatientCareSetting
pastOrders: pastOrdersReducer.pastOrders,
});
```

Reply:
Will be fixed in @fuchodeveloper 's PR for the careSetting